cockroachcloud/managing-cmek.md

```diff
@@ -236,11 +234,104 @@ To rotate the CMEK keys for one or more cluster regions:
       --data "@cmek_config.json"
     ```
 
+## Add a region to a CMEK-enabled cluster
+
+To add a region to a cluster that already has CMEK enabled, update your cluster's region definitions using the {{ site.data.products.db }} API [Update Cluster](https://www.cockroachlabs.com/docs/api/cloud/v1#operation/CockroachCloud_UpdateCluster) endpoint.
+
+1. Construct the data payload JSON to update you region definitions.
+
+    The payload format depends on which cloud provider you use:
+    - **AWS**:
+        {% include_cached copy-clipboard.html %}
+        ```json
+        # cmek_aws_config.json
+        {
+            "dedicated": {
+                "region_nodes": {
+                    "us-west-1": 3,
+                    "us-central-1": 5
+                },
+                "cmek_region_specs": [
+                    {
+                        "region": "us-west-1",
+                        "key_spec": {
+                            "type": "AWS_KMS",
+                            "uri": "{id-of-key}",
+                            "auth_principal": "{role-with-kms-access}"
+                        }
+                    },
+                    {
+                        "region": "us-central-1",
+                        "key_spec": {
+                            "type": "AWS_KMS",
+                            "uri": "{id-of-another-key}",
+                            "auth_principal": "{another-role-with-kms-access}"
+                        }
+                    }
+                ],
+                "hardware": {
+                    "machine_type": "n2-standard-8"
+                }
+            }
+        }
+        ```
+    - **GCP**:
+        {% include_cached copy-clipboard.html %}
+        ```json
+        # cmek_gcp_config.json
+        {
+            "dedicated": {
+                "region_nodes": {
+                    "us-west1": 3,
+                    "us-central1": 5
+                },
+                "cmek_region_specs": [
+                    {
+                        "region": "us-west1",
+                        "key_spec": {
+                            "type": "GCP_KMS",
```

Comment:
nit: the type should be GCP_CLOUD_KMS. yes, this is not parallel with AWS_KMS; IIRC we decided to use the official names of the products in each cloud (which is just KMS for AWS, and cloud KMS for GCP)